src/git.md

```diff
@@ -263,6 +264,67 @@ You also may want to squash just the last few commits together, possibly
 because they only represent "fixups" and not real changes. For example,
 `git rebase --interactive HEAD~2` will allow you to edit the two commits only.
 
+### `git range-diff`
+
+After completing a complicated rebase, or even a relatively simple one, you may
+want to review the changes between your old branch and your new one. You can do
+that with the `git range-diff master @{upstream} HEAD` command, which can be
+very helpful when you had a complicated rebase and you want to make sure you
+changed the right things.
```

Comment:
```suggestion
that with `git range-diff master @{upstream} HEAD`.
```

Reply:
How about something like this:

```suggestion
that with the `git range-diff master @{upstream} HEAD` command, which can help
you make sure that you merged the upstream changes correctly.
```

Reply:
I don't see what that adds? We already said "you may want to review the changes" above.

Reply:
Yeah, I see what you mean 👍